pom.xml

```diff
@@ -80,23 +80,23 @@
     <dependency>
       <groupId>com.avispl.symphony.api</groupId>
```

Comment:
please update httpclient, junit versions, that provoke the dependabot warnings, also please align other dependency versions based on "3rd Party Software - Symphony" confluence page